### ISSUE TITLE
mysql support

### ISSUE THREAD

**Issue by SleepyAsh0191** [tritonmc/triton-collection-generator#40]
just need support for export to .sql file not the json

(although i wrote a simple script to convert)

**Comment by diogotcorreia** [comment]
I don't think it makes sense for this tool to export SQL files. I'd rather have features to more easily import this JSON in both Triton and TWIN :thinking: 

I'll keep this open because the underlying problem still exists: it's rather difficult to import the resulting files from this tool if you're using the SQL storage options. The (designed) workaround right now is to use the `/triton database` command (but it's rather destructive and must be used with care).
